src/stateless/selectbox.vue

```diff
@@ -432,6 +460,10 @@ export default {
     .selectbox__label-text {
         color: @bluish-gray;
 
+        &--disabled {
```

Comment:
Did we have the disabled state broken on master or why did we need to add this?

Reply:
Yep. It's not used anywhere so we didn't notice it.

Reply:
:+1: